src/js/media-captions-listbox.js

```diff
@@ -91,15 +94,15 @@ class MediaCaptionsListbox extends MediaChromeListbox {
       this.#render();
 
     } else if (attrName === MediaUIAttributes.MEDIA_SUBTITLES_SHOWING && oldValue !== newValue) {
-      const selectedTrack = parseTextTrackStr(newValue ?? '');
+      const selectedTrack = parseTextTracksStr(newValue ?? '')[0];
```

Comment:
shame TS didn't catch this. I would think it should have with the props `.language` and `.label` later on

Reply:
TS wouldn't catch this, since it's a behavioral thing.